index.js

```diff
@@ -0,0 +1,162 @@
+
+const section = document.querySelector('.container');
+const mainForm = document.getElementById('book-form');
+const displayPage = document.querySelector('.display-page');
+
+const navBar = document.createElement('ul');
+const navMenuDiv = document.createElement('div');
+navMenuDiv.className = 'nav-menu-div';
+navBar.className = 'nav-bar';
+const navHeading = document.createElement('h2');
+navHeading.textContent = 'Awesome Books';
+navHeading.className = 'nav-title';
+const listMenu = document.createElement('li');
+listMenu.id = 'first-list';
+const listLink = document.createElement('a');
+listLink.textContent = 'List';
+listLink.className = 'menu-link';
+listMenu.append(listLink);
+navMenuDiv.append(listMenu);
+
+const addMenu = document.createElement('li');
+addMenu.id = 'second-list';
+const addLink = document.createElement('a');
+addLink.textContent = 'Add Book';
+addLink.className = 'menu-link';
+addMenu.append(addLink);
+navMenuDiv.append(addMenu);
+
+const contactMenu = document.createElement('li');
+contactMenu.id = 'third-list';
+const contactLink = document.createElement('a');
+contactLink.textContent = 'Contact';
+contactLink.className = 'menu-link';
+contactMenu.append(contactLink);
+navMenuDiv.append(contactMenu);
+navBar.append(navMenuDiv);
+navBar.prepend(navHeading);
+const currentDate = DateTime.now();
+
+document.querySelector('.time').innerHTML = currentDate.toLocaleString(DateTime.DATETIME_MED);
+
+class Book {
+  constructor() {
+    this.bobject = {};
+    this.list = [];
+  }
+
+  add(title, author) {
+    this.bobject = { title, author };
+    this.list.push(this.bobject);
+    localStorage.setItem('data', JSON.stringify(this.list));
+  }
+
+  remove(getId) {
+    let new1 = 0;
+    while (new1 < this.list.length) {
+      if (new1.toString() === getId) {
+        this.list.splice(new1, 1);
+        localStorage.setItem('data', JSON.stringify(this.list));
+      }
+
+      new1 += 1;
+    }
+  }
+}
+
+const book = new Book();
+
+const bookList = document.createElement('ul');
+bookList.style.marginTop = '20px';
+
+bookList.className = 'book-list';
+displayPage.append(bookList);
+const anyRandomNAme = () => {
+  book.list.forEach((each, bookId) => {
+    const list1 = document.createElement('li');
+    list1.className = 'list';
+
+    const title = document.createElement('h2');
+    title.innerHTML = `"${each.title}"`;
+    const phrase = document.createElement('b');
+    phrase.textContent = 'By';
+    list1.appendChild(title);
+    list1.appendChild(phrase);
+
+    const author = document.createElement('h2');
+    author.innerHTML = each.author;
+    list1.appendChild(author);
+
+    const removeButton = document.createElement('button');
+    removeButton.className = 'remove';
+    removeButton.id = `${bookId}`;
+    removeButton.innerHTML = 'Remove';
+    list1.appendChild(removeButton);
+
+    bookList.appendChild(list1);
+    removeButton.addEventListener('click', function removeBtnHandler() {
```

Comment:
- [ ] Since you are using ES6 I would recommend using `arrow` function instead of using the traditional function in order to make your project follow the project requirement :smile:

Reply:
i dont understand your intention here please elaborate

Reply:
Hi @vigehi the purpose of this milestone is to follow ES6 best practices you did well so far but for this part you are using regular function instead of arrow function which is required :+1: